CHANGELOG.md

```diff
@@ -5,6 +5,10 @@
  - properly support setting host when registering task set 
  - rename `response` wrapper to `goose`, so we end up with `goose.request` and `goose.response`
  - add `--throttle-requests` to optionally limit the maximum requests per second (api change)
+ - change task function signature, tasks must return a GooseTaskResult
+ - introduce `GooseTaskError`
```

Comment:
Ah - this belongs to the previous PR?

I would probably make a quick commit / PR outside of this one as it's confusing ;)

Reply:
Ok, there's also a tiny README change that was also missed in the previous PR, I'll merge both outside of this new PR